portfolio/src/main/webapp/index.html

```diff
@@ -38,6 +38,10 @@ <h1>Jesús Urquídez Calvo</h1>
     <p>Click here to get a random message:</p>
     <button onclick="addRandomGreeting()">Hello</button>
     <div id="greeting-container"></div>
+    <button onclick="callServlets();">
+      Random Number from 0 to 9
+    </button>
+    <p id="date-container"></p>
```

Comment:
Can you update the ID here and wherever else it's referenced to be more specific, like `random-number-container`?

Reply:
okay